pkg/controller/pod_volume_backup_controller.go

```diff
@@ -101,8 +101,6 @@ func (r *PodVolumeBackupReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		)
 	}
 
-	log.Info("PodVolumeBackup starting")
```

Comment:
Maybe it's more suitable to  move this log to L116, only "", `velerov1api.PodVolumeBackupPhaseNew` will print this log

Reply:
Thanks.
Make sense to me.
Rebased.